v2/css/xocolor.css

```diff
@@ -1,6 +1,6 @@
 .xo-color256 { --stroke-color: #666666; --fill-color: #ffffff; }
 .xo-color512 { --stroke-color: #000000; --fill-color: #999999; }
-.xo-color768 { --stroke-color: #ffffff; --fill-color: #000000; }
+.xo-color768 { --stroke-color: #ffffff; --fill-color: transparent; }
```

Comment:
Not a big fan of this fix. Will be better to hard set the color in the SVG.

Reply:
could you please elaborate on what you mean by hard set the color in SVG?

Reply:
![image](https://github.com/llaske/sugarizer/assets/98237998/68cc88e3-872a-49f8-ac8c-08632e99b4c5)

![image](https://github.com/llaske/sugarizer/assets/98237998/bfff3617-6fb9-4a2b-9381-46d81ab338b7)

is this fix any better?